### PR TITLE
FEAT: Added Auth and Responsiveness

### PR DESCRIPTION
**Pull request by fahd23**
- Added Auth functionality
- Added Responsiveness
- Removed Forgot Password Page
- Refactored the code

Preview Link: https://deploy-preview-10--cricbazzar-react.netlify.app/

### CODE REVIEW

**Review by JantuDeb — APPROVED** [review]
This is looking good

File: src/components/cart/Cart.jsx [hunk 1]
```diff
@@ -25,146 +28,145 @@ const Cart = () => {
...
+                      </button>
+                      <button
+                        className="btn otld-default"
+                        disabled={wishlist.find(
+                          (item) => item._id === product._id
+                        )}
+                        onClick={() => {
+                          wishlistDispatch({
+                            type: "ADD_TO_WISHLIST",
+                            payload: product,
```
Comment (tusharkandpal) [inline src/components/cart/Cart.jsx:112]: `payload` is preferred to be an object

File: src/components/navbar/Navbar.jsx [hunk 2]
```diff
@@ -1,69 +1,134 @@
 import "components/navbar/navbar.css";
+import { useAuth } from "context/auth-context";
 import { useCart } from "context/cart-context";
+import { useProduct } from "context/product-context";
 import { useWishlist } from "context/wishlist-context";
```
Comment (tusharkandpal) [inline src/components/navbar/Navbar.jsx]: can use barrel exports for these.

**Review by tusharkandpal — APPROVED** [review]
code looks good ! 

suggestion: for resetting wishlist & cart you can set initial state again. No need to send `payload` for it.

**Pull request merged**